Minor improvement to the nested aggregation docs (#46475)
elastic/elasticsearch: docs/reference/aggregations/bucket/nested-aggregation.asciidoc

[source,console]
--------------------------------------------------
PUT /products
{
    "mappings": {
        "properties" : {
            "resellers" : { <1>
                "type" : "nested",
                "properties" : {
                    "reseller" : { "type" : "text" },
                    "price" : { "type" : "double" }
                }
            }
        }
    }
}
--------------------------------------------------

> <1> The `resellers` is an array that holds nested documents.

Reviewer: ```suggestion
<1> `resellers` is an array that holds nested documents.
```

> We are adding one example product with two resellers:

Reviewer: ```suggestion
The following request adds a product with two resellers:
```

[source,console]
--------------------------------------------------
PUT /products/_doc/0
{
  "name": "LED TV", <1>
  "resellers": [
    {
      "reseller": "companyA",
      "price": 350
    },
    {
      "reseller": "companyB",
      "price": 500
    }
  ]
}
--------------------------------------------------
// TEST[s/PUT \/products\/_doc\/0/PUT \/products\/_doc\/0\?refresh/]
// TEST[continued]
<1> We are using a dynamic mapping for the `name` attribute.

> The following aggregations will return the minimum price a product can be purchased for:

Reviewer: ```suggestion
The following request returns the minimum price a product can be purchased for:
```
